test/e2e/e2e_test.go

```diff
@@ -88,7 +88,16 @@ func TestEventTransformationForSubscription(t *testing.T) {
 	}
 	cancel := logstream.Start(t)
 	defer cancel()
-	e2ehelpers.EventTransformationForSubscriptionTestHelper(context.Background(), t, e2ehelpers.SubscriptionV1beta1, channelTestRunner)
+	e2ehelpers.EventTransformationForSubscriptionTestHelper(context.Background(), t, e2ehelpers.SubscriptionV1beta1, channelTestRunner,
```

Comment:
I think the effect is the same as https://github.com/google/knative-gcp/pull/1695#discussion_r489987937?

Reply:
Yes, but I think the signature changed, and probably that is why they removed it. 
There are a bunch of e2e tests that are currently skipped and we should start unskipping them. The ones you mentioned, I think they were all skipped, that is why nobody noticed the missing secret